### PR TITLE
Move PostgreSQL Operator to samples repo

### PR DESCRIPTION
**Pull request by pmacik**
This PR:
* Reflects moving the PostgreSQL Operator to https://github.com/operator-backing-service-samples/postgresql-operator repo.

### CODE REVIEW

File: vendor/github.com/operator-backing-service-samples/postgresql-operator/pkg/apis/postgresql/v1alpha1/database_types.go [hunk 1]
```diff
@@ -28,8 +28,9 @@ type DatabaseStatus struct {
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 	DBCredentials    string `json:"dbCredentials"`
 	DBConnectionIP   string `json:"dbConnectionIP"`
-	DBConnectionPort int32  `json:"dbConnectionPort"`
+	DBConnectionPort int64  `json:"dbConnectionPort"`
 	DBName           string `json:"dbName"`
+	DBConfigMap      string `json:"dbConfigMap"`
```
Comment (DhritiShikhar) [inline vendor/github.com/operator-backing-service-samples/postgresql-operator/pkg/apis/postgresql/v1alpha1/database_types.go]: Why is this change a part of moving PostgreSQL operator PR?

Comment (pmacik) [reply vendor/github.com/operator-backing-service-samples/postgresql-operator/pkg/apis/postgresql/v1alpha1/database_types.go]: It is likely that the operator code cached in the original vendor directory was not up to date... The new content is definitely correct.

**Pull request merged**